awx/ui/src/screens/Setting/MiscSystem/MiscSystemDetail/MiscSystemDetail.js

```diff
@@ -63,6 +67,10 @@ function MiscSystemDetail() {
       Object.keys(systemData).forEach((key) => {
         mergedData[key] = options[key];
         mergedData[key].value = systemData[key];
+
+        if (key === 'AUTOMATION_ANALYTICS_LAST_ENTRIES') {
+          mergedData[key].value = formatJson(systemData[key]) ?? '';
```

Comment:
We could avoid re-write the value to mergedData.

```        mergedData[key].value =
          key === 'AUTOMATION_ANALYTICS_LAST_ENTRIES'
            ? (mergedData[key].value = formatJson(systemData[key]) ?? '')
            : systemData[key];
      });```

But this is less readable IMO.